deploy/templates/nstemplatetiers/appstudio/spacerole_admin.yaml

```diff
@@ -242,6 +241,41 @@ objects:
   - kind: User
     name: ${USERNAME}
 # RoleBinding that grants view permissions to the User
+
+# Role(s) and RoleBinding(s) that grant CRUD permissions on SecretStore and ExternalSecret CRs to the user's SA
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: Role
+  metadata:
+    namespace: ${NAMESPACE}
+    name: appstudio-admin-user-actions-externalsecrets
+  rules:
+  - apiGroups:
+    - external-secrets.io/v1beta1
+    resources:
+    - SecretStore
+    - ExternalSecret
```

Comment:
I see all the other existing resources in the `appstudio-admin-user-actions`  Role are all lower case and plural form. Should we be consistent ?

Reply:
Yes, the resource names must be in plural form and lower case.

Reply:
Updated resource names to be plural form and lower case.